Log to stdout in addition to file

gidsg opened the pull request:
https://trello.com/c/VZBPctdG/106-2-our-apps-do-not-log-to-stdout-timebox-max-2-days

N.B. This should be tested on Preview to verify it is working as expected on PaaS.

katstevens commented on dmutils/logging.py:
```diff
@@ -85,10 +85,21 @@ def after_request(response):
         return response
 
     logging.getLogger().addHandler(logging.NullHandler())
+    handlers = [logging.StreamHandler(sys.stdout)]
```
Do we think this should be configurable per-app, like the other handler?

gidsg replied:
@katstevens can we leave as-is and if we find a use case to make it configurable per-app we could always add it later?

katstevens replied:
Sounds reasonable 😸

gidsg commented:
> could we update the reference to the now-deleted get_handler function in the tests.conftest.app_with_stream_logger fixture?

@katstevens great spot. I updated this now in f9c0c48.
